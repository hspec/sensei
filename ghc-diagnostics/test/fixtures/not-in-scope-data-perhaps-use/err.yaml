version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/not-in-scope-data-perhaps-use/Foo.hs
  start:
    line: 3
    column: 7
  end:
    line: 3
    column: 17
severity: Error
code: 88464
message:
- 'Variable not in scope: someOption'
hints:
- Perhaps use data constructor `SomeOption' (line 2)
