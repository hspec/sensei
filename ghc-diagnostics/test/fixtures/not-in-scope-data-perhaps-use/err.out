test/fixtures/not-in-scope-data-perhaps-use/Foo.hs:3:7: error: [GHC-88464]
    Variable not in scope: someOption
    Suggested fix: Perhaps use data constructor `SomeOption' (line 2)

